pihole/metadata.csv

```diff
@@ -0,0 +1,15 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+queries_forwarded,gauge,,query,,Queries not blocked,0,Pihole,pihole.queries_forwarded
```

Comment:
This should be the other way around. `metric_name` should have the `pihole` prefix while `short_name` don't